### DISCUSSION TITLE
Asking Questions

### DISCUSSION THREAD

**Discussion by eva-sui** [ion-oset/developer-commons#29] [Q&A]
how to ask question @stratofax @cwulfman @ion-oset @WISSAL-MN https://github.com/github

**Comment by stratofax** [comment]
Hey, @eva-sui I saw in my GitHub notifications that you got some excellent advice elsewhere. To summarize: 

- be specific, provide examples, especially input and output, and configuration info, like software or package versions.
- I'd add to that: consider the context of your question. Take some time to learn what the goal of the repo is, and try to keep your question relevant to the overall purpose of the repo when you post in its discussion boards.
- Choose a descriptive subject line for your topic so others can see what it's about without having to read your entire post. For example, I renamed your post from "test" to "Ascking Questions"
- Also, read what other people have posted -- try not to ask the same question over and over again. 
- It's not good idea to "@" someone for a general question, like yours. If you don't "@" anyone, it encourages others to chime in, without demanding the attention of repo admins and maintainers, who are often busy with other development tasks.

Having said that, I appreciate your question because I realize that we haven't been very clear about our guidelines for this repo. Just so you know, this repo describes best practices for [The TrustTheVote® Project](https://github.com/TrustTheVote-Project). 

We're making this repo public with the hopes that our open source development experiences and best practices can help other developers too, especially devs who want to contribute to this project. 

In that spirit, we'd all appreciate if everyone who participates in our discussions does so with a genuine interest in advancing the work represented here and across the TrustTheVote project. 

If you have questions that aren't specifically about the TrustTheVote project, we probably won't be able to answer them in a timely manner, if at all. In this case, I'm happy to answer your question because it's helped me to articulate the policies of this discussion board. 

I hope you find the other resources we've posted here informative and useful.

**Reply by eva-sui** [reply]
@stratofax  I understand thank you very much
